### PR TITLE
Use custom icons/names for leak canary

### PR DESCRIPTION
**Pull request by jonathan-caryl**
If you have a bunch of different apps installed that all use LeakCanary, it's hard to tell what's what, as they all use the same name/icon in the launcher. But you can customise it for each app.

leak_icons

### CODE REVIEW

File: sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java [hunk 1]
```diff
@@ -18,6 +18,11 @@
 
   @Override public void onCreate() {
     super.onCreate();
+    if (LeakCanary.isInAnalyzerProcess(this)) {
```
Comment (jonathan-caryl) [inline sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java]: This had been added to `sample-mvi` but not to the other samples.

Comment (sockeqwe) [inline sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java]: Yeah, my bad. This should be added to all examples that use LeakCanary. Could you please add that to the other examples too?

**Comment by sockeqwe** [comment]
Nice! That looks awesome. Thank you very much!

File: sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java (hunk 1 above)
Comment (sockeqwe) [inline sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java]: Arrrr ... have overlooked that you have already added it!
Forget about my last comment!

Thanks a lot!

**Pull request merged**